spring-kafka/src/main/java/org/springframework/kafka/aot/KafkaRuntimeHints.java

```diff
@@ -200,15 +142,10 @@ public void registerHints(RuntimeHints hints, @Nullable ClassLoader classLoader)
 
 		Stream.of(
 				"sun.security.provider.ConfigFile",
-				"org.apache.kafka.streams.processor.internals.StreamsPartitionAssignor",
-				"org.apache.kafka.streams.errors.DefaultProductionExceptionHandler",
-				"org.apache.kafka.streams.processor.FailOnInvalidTimestamp",
-				"org.apache.kafka.streams.processor.internals.assignment.HighAvailabilityTaskAssignor",
 				"org.apache.kafka.streams.processor.internals.assignment.StickyTaskAssignor",
 				"org.apache.kafka.streams.processor.internals.assignment.FallbackPriorTaskAssignor",
```

Comment:
Do we really still need to keep these two, as well?

Reply:
ya, they are missing from the metadata repo. Once they are added to the metadata, then we can remove them.

Reply:
So, all good. Leaving as is for now.